packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -74,10 +74,7 @@ export class LitAuthClient {
     }
 
     // Check if Lit node client is provided
-    if (
-      options?.litNodeClient &&
-      options.litNodeClient instanceof LitNodeClient
-    ) {
+    if (options?.litNodeClient) {
```

Comment:
i had to remove the instanceOf check, because it was always failing, even when i passed in the litNodeClient.  removing this fixed it.